horde/apis/v2/base.py

```diff
@@ -3128,3 +3150,103 @@ def get(self):
         if self.args.format == "markdown":
             return {"markdown": markdownify(html_template).strip("\n")}, 200
         return {"html": html_template}, 200
+
+
+@dataclass
+class KnownModelsReturn:
+    name: str
+    type: str
+    metadata: KnownModelRef
+
+
+def known_models(model_type: str = None, filter_model_name: str = None, exact: bool = None) -> list[KnownModelsReturn]:

```

Comment:
This should go inside the `model_reference.py` instead of at the root of the API file.